gensim/models/ensemblelda.py

```diff
@@ -0,0 +1,1315 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Authors: Tobias Brigl <github.com/sezanzeb>, Alex Salles <alex.salles@gmail.com>,
+# Alex Loosley <aloosley@alumni.brown.edu>, Data Reply Munich
+#
+
+
+"""Ensemble Latent Dirichlet Allocation (eLDA), a method of training a topic model ensemble.
+
+Extracts reliable topics that are consistently learned across multiple LDA models. eLDA has the added benefit that
+the user does not need to know the exact number of topics the topic model should extract ahead of time
+
+For more details read our paper (https://www.hip70890b.de/machine_learning/ensemble_LDA/).
+
+Usage examples
+--------------
+
+Train an ensemble of LdaModels using a Gensim corpus
```

Comment:
```suggestion
Train an ensemble of LdaModels using a Gensim corpus.
```

Either full stop `.` or colon `:` at the end of sentences please.

(last such comment, but I see many more instances)